src/cascadia/TerminalApp/App.cpp

```diff
@@ -215,11 +215,49 @@ namespace winrt::TerminalApp::implementation
     void App::_CreateNewTabFlyout()
     {
         auto newTabFlyout = Controls::MenuFlyout{};
+        auto _keyBindings = _settings->GetKeybindings();
+
         for (int profileIndex = 0; profileIndex < _settings->GetProfiles().size(); profileIndex++)
         {
             const auto& profile = _settings->GetProfiles()[profileIndex];
             auto profileMenuItem = Controls::MenuFlyoutItem{};
 
+            // add the keyboard shortcuts for the first 9 profiles
+            if (profileIndex < 9)
+            {
+                // enum value for ShortcutAction::NewTabProfileX starts at 3; 0==3
+                auto profileKeyChord = _keyBindings.GetKeyBinding(static_cast<ShortcutAction>(profileIndex + 3));

```

Comment:
Instead of `3` I might prefer _cast `NewTabProfile0` (or whatever the base value is) to an integer_; thoughts?

Reply:
If that is perceived to be better, that's an easy one...